packages/scripts/src/generate-master-list/generator.ts

```diff
@@ -139,22 +140,20 @@ const getPackage = (adapterPath: string, verbose = false) => {
 
     if (packageJson.dependencies) {
       let dependencyList = Object.keys(packageJson.dependencies)
-
       dependencyList = dependencyList.reduce((list: string[], dep) => {
         const depSplit = dep.split('/')
-        if (depSplit[0] === '@chainlink' && !baseEaDependencies.includes(depSplit[1]))
-          list.push(wrapCode(depSplit[1]))
+        if (depSplit[0] === '@chainlink' && !baseEaDependencies.includes(depSplit[1])) []
+        list.push(wrapCode(depSplit[1]))
+        if (depSplit[1] === 'external-adapter-framework') frameworkVersion = 'v3'
```

Comment:
You probably want to check the entire string, otherwise a different "npm org" with the same repo name would trigger this.

Reply:
done